tests/test_route53/test_route53_cloudformation.py

```diff
@@ -1,8 +1,8 @@
-import boto3
 import json
-import sure  # noqa # pylint: disable=unused-import
-
 from copy import deepcopy
+
+import boto3
+import sure  # noqa # pylint: disable=unused-import
```

Comment:
Hi @giograno, we've removed the `sure`-module a while back - might be worth merging in the latest master to make sure you're working with the latest version.

Reply:
Hi @bblommers, I am sorry! Wasn't on the latest master. I just rebased my PR.